criu/mount.c

```diff
@@ -2472,6 +2477,14 @@ static int do_close_one(struct mount_info *mi)
 	return 0;
 }
 
+static int set_unbindable(struct mount_info *mi)
+{
+	if (mi->flags & MS_UNBINDABLE && !mnt_is_overmounted(mi)) {
+		return mount(NULL, mi->mountpoint, NULL, MS_UNBINDABLE, NULL);
```

Comment:
we need to print an error message if mount fails. I see that we already have this problem in our code, so it would be good to fix them too.